code/modules/power/antimatter/control.dm

```diff
@@ -108,14 +110,16 @@
 				toggle_power()
 			stability -= rand(10,20)
 	..()
-	return 0
+	return FALSE
 
 
 /obj/machinery/power/am_control_unit/blob_act()

```

Comment:
add the default param, `obj/structure/blob/B`, then correct the AMS.blob_act calls below